get-started/nav.html

```diff
@@ -1,7 +1,7 @@
 <ul class="pagination">
-  <li {% if include.selected=="1"%}class="active"{% endif %}><a href="part1">1: Orientation and setup</a></li>
-  <li {% if include.selected=="2"%}class="active"{% endif %}><a href="part2">2: Containerizing an application</a></li>
-  <li {% if include.selected=="3"%}class="active"{% endif %}><a href="part3">3: Deploying to Kubernetes</a></li>
-  <li {% if include.selected=="4"%}class="active"{% endif %}><a href="part4">4: Deploying to Swarm</a></li>
-  <li {% if include.selected=="5"%}class="active"{% endif %}><a href="part5">5: Sharing images on Docker Hub</a></li>
+  <li {% if include.selected=="1"%}class="active"{% endif %}><a href="part1">Orientation and setup</a></li>
+  <li {% if include.selected=="2"%}class="active"{% endif %}><a href="part2">Containerizing an Application</a></li>
+  <li {% if include.selected=="3"%}class="active"{% endif %}><a href="part3">Deploying to Kubernetes</a></li>
+  <li {% if include.selected=="4"%}class="active"{% endif %}><a href="part4">Deploying to Swarm</a></li>
+  <li {% if include.selected=="5"%}class="active"{% endif %}><a href="part5">Sharing Images on Docker Hub</a></li>
```

Comment:
We generally use sentence case for document titles, so `application` and `images` should both remain lowercase, both here and in the titles of the pages this file is referring to.

Reply:
yep fixed thank you